include/lsqpack.h

```diff
@@ -473,8 +473,8 @@ struct lsqpack_enc
 
         /* Number of at-risk references in this header block */
         unsigned            n_risked;
-        /* Number of headers in this header list */
-        unsigned            n_headers;
+        /* Number of headers in this header list that are in the history */
```

Comment:
That's not quite true: the "add" function was called, but there is no guarantee that they are in history.  How about something like "number of headers added to history"

Reply:
I see. I will update the name to address this.